src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/PrisonResourceTest.kt

```diff
@@ -0,0 +1,44 @@
+package uk.gov.justice.hmpps.prison.api.resource.impl
+
+import org.junit.jupiter.api.Test
+import org.springframework.core.ParameterizedTypeReference
+import org.springframework.http.MediaType
+import uk.gov.justice.hmpps.prison.api.model.SentenceSummary
+
+class PrisonResourceTest : ResourceTest() {
+
+  @Test
+  fun testPrisonBookingSummaryReturnsListResponseReturnsSentenceSummaryList() {
```

Comment:
You could use back ticks to make this name read better. Can't tell if it makes sense or not 😄

Reply:
Mike I've addressed this one :)